app/views/paye/paye_your_income_and_taxes.scala.html

```diff
@@ -24,7 +24,11 @@
 @(viewModel:PayeYourIncomeAndTaxes)(implicit lang: Lang, messages: Messages, request: PayeAuthenticatedRequest[_], formPartialRetriever: FormPartialRetriever)
 
 @headerContent = {
-
+  @if(lang.language == "cy"){
```

Comment:
Is this toggle not already in the paye main template?

Reply:
This toggle is not part of main template.
I understand the confusion as In the last PR it was added on the main landing page not on the main template.

Reply:
As discussed toggle changes are moved to main template now .